packages/elements/src/photon-multirx-form/components/AddPrescriptionCard.tsx

```diff
@@ -314,34 +316,7 @@ export const AddPrescriptionCard = (props: {
                     'notes'
                   ]);
                   setOffCatalog(undefined);
-                  props.actions.updateFormValue({
```

Comment:
@jknapp25 good callout on the init form values, I couldn't find it but I did find some duplicate code that could use the new function :)